### ISSUE TITLE
`language` can only be set from the config file

### ISSUE THREAD

**Issue by johnnovak** [dosbox-staging/dosbox-staging#4080]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.83.0-alpha

### What Operating System are you using?

macOS

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

`language` cannot be set at runtime, followed by a reboot via `config -r`, and the more interesting thing, it cannot be even set with `--set language=XY` from the CLI.

Not exactly a showstopper or a priority, but a bug, nevertheless 😄 

### Steps to reproduce the behaviour.

- Setting `language = fr` (or any other non-English language) in the conf file works. 🍏 

- `dosbox --noprimaryconf --set "language = fr"` doesn't work (English gets picked, but executing `language` prints out `fr`). 🛑 

- Running `language fr` in the CLI followed by `config -r` to restart doesn't work  (English gets picked, but executing `language` prints out `fr`). 🛑 

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by FeralChild64** [comment]
I intend to do this eventually. On the current `main` branch there are two problems:

1. The `language` setting is a part of `dosbox` section, which is set as read-only. Of course we can handle it one way or another.
2. The current implementation in `messages.cpp` forgets the English strings added by `MSG_Add` when it loads the translation, which means we can't revert. I have already reworked this code, PR is ready for review: https://github.com/dosbox-staging/dosbox-staging/pull/4074 - once it is merged, we can proceed with with this issue.

**Comment by johnnovak** [comment]
> I have already reworked this code, PR is ready for review: https://github.com/dosbox-staging/dosbox-staging/pull/4074 - once it is merged, we can proceed with with this issue.

@FeralChild64 That was in November 😆 Just wondering, are you going to do this or is it even still an issue?

**Comment by FeralChild64** [comment]
Yes, I still intend to do this - so far I was busy with other issues/features, or just didn’t have time to work on the DOSBox.
